src/ApiDecider.php

```diff
@@ -35,11 +35,12 @@ public function getApi(string $method, int $version, string $package, ?string $a
 
         foreach ($this->apis as $api) {
             $identifier = $api->getEndpoint();
-            if ($method === $identifier->getMethod() && $identifier->getVersion() === $version && $identifier->getPackage() === $package && $identifier->getApiAction() === $apiAction) {
-                $endpointIdentifier = new EndpointIdentifier($method, $version, $package, $apiAction);
+            $endpointIdentifier = new EndpointIdentifier($method, $version, $package, $apiAction);
```

Comment:
ak pride $apiAction = '' tak equal nevrati true nikdy, lebo v EndpointIdentifier-e je if na to - $apiAction === '' ? return null : $apiAction;